app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkChaptersCompletedFragmentTest.kt

```diff
@@ -422,6 +423,58 @@ class MarkChaptersCompletedFragmentTest {
     }
   }
 
+  @Test
+  fun testMarkChaptersCompletedFragment_unSelectAll_unSelectsAllChapters() {
+    launch<MarkChaptersCompletedTestActivity>(
+      createMarkChaptersCompletedTestActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.mark_chapters_completed_all_check_box_container)).perform(click())
+      scrollToPosition(position = 1)
```

Comment:
Could we maybe include the scroll within the helper? It'd cut down a lot on the boilerplate required in the test.

Reply:
This has been clarified in another comment, and the latest changes address my original comment here.